### PR TITLE
feat(bulk-create): add connect-to feature

### PR DESCRIPTION
**Pull request by LawrenceBorst**
Fixes Lundalogik/crm-feature#3329

## Review:
- [ ] Commits are [atomic](https://seesparkbox.com/foundry/atomic_commits_with_git)
- [ ] Commits have the correct *type* for the changes made
- [ ] Commits with *breaking changes* are marked as such

### Browsers tested:
(Check any that applies, it's ok to leave boxes unchecked if testing something didn't seem relevant.)

Windows:
- [ ] Chrome
- [ ] Edge
- [ ] Firefox

Linux:
- [ ] Chrome
- [ ] Firefox

macOS:
- [ ] Chrome
- [ ] Firefox
- [ ] Safari

Mobile:
- [ ] Chrome on Android
- [ ] iOS

### CODE REVIEW

File: src/components/list/list-item.types.ts [hunk 1]
```diff
@@ -45,6 +45,11 @@ export interface ListItem<T = any> {
      * Component used to render in the list
      */
     primaryComponent?: ListComponent;
+
+    /**
+     * True if the list item should be removable (only works for limel-chips)
+     */
+    removeable?: boolean;
```
Comment (jgroth) [inline src/components/list/list-item.types.ts:52]: I'm not sure this makes much sense at the moment from the lists point of view, so I don't think we should add this here. Also, I'm not sure about the description, how is this connected to the chips? It seems to be used by the picker and the `Chip` interface :thinking:

**Review by jgroth — COMMENTED** [review]
A comment about the commit message, Lime Elements does not know what "bulk create" is :smile:

File: src/components/list/list-item.types.ts (hunk 1 above)
Comment (LawrenceBorst) [reply src/components/list/list-item.types.ts:52]: Ah good point! I double checked, it seems we could instead pass in the `removable` (I misspelled it in the commit 💀 ) as a prop into the `limel-picker` in `relation-picker.tsx`. The only reason we need it is for `createChip()` in `picker.tsx`, and I just blindly added it to the interface

Comment (LawrenceBorst) [reply src/components/list/list-item.types.ts:52]: Looked into it more

We can close this PR, all the changes can be handled by just passing in`readonly=true`

**Pull request closed without merging**